### PR TITLE
Integrate avalanche

### PR DESCRIPTION
**Pull request by MrQKyber**

### CODE REVIEW

File: KyberNetwork/Settings/Types/Constants.swift [hunk 1]
```diff
@@ -143,10 +145,36 @@ public struct Constants {
...
+    ensAddress: "",
+    wrappedAddress: "",
+    apiEtherscanEndpoint: ""
+  )
+  
+  public static let avalancheMainnetPRC = CustomRPC(
+    chainID: 43114,
+    name: "Avalanche Mainnet C-Chain",
+    symbol: "AVAX",
+    endpoint: "https://speedy-nodes-nyc.moralis.io/847df1933775fb519982918b/avalanche/mainnet/",
```
Comment (toidv) [inline KyberNetwork/Settings/Types/Constants.swift:165]: Please use this one as primary node: https://api.avax.network/ext/bc/C/rpc. The node https://speedy-nodes-nyc.moralis.io/847df1933775fb519982918b/avalanche/mainnet/ is just temporary solution

File: KyberNetwork/KyberNetwork/Networking/External/KNGeneralProvider.swift [hunk 2]
```diff
@@ -234,6 +267,8 @@ class KNGeneralProvider {
       return "There.is.a.difference.between.the.estimated.price.bsc"
     case .polygon:
       return "There.is.a.difference.between.the.estimated.price.matic"
+    case .avalanche:
+      return "There.is.a.difference.between.the.estimated.price.matic" //TODO: wait for avalance
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Networking/External/KNGeneralProvider.swift:271]: Please change it accordingly to `Coingecko, and Pangolin`

File: KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/Controllers/EarnMenuViewController.xib [hunk 3]
```diff
@@ -9,7 +9,7 @@
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <customFonts key="customFonts">
-        <array key="Karla-Regular.ttf">
+        <array key="Karla-VariableFont_wght.ttf">
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/Controllers/EarnMenuViewController.xib:12]: It is correct font?

File: KyberNetwork/KyberNetwork/Screens/Tabbars/KyberSwap/SwapVC/Coordinator/KNExchangeTokenCoordinator.swift [hunk 4]
```diff
@@ -158,7 +158,8 @@ extension KNExchangeTokenCoordinator {
     self.navigationController.popToRootViewController(animated: true)
     let otherToken: TokenObject = token.isETH ? KNSupportedTokenStorage.shared.kncToken : KNSupportedTokenStorage.shared.ethToken
     let otherTokenBsc: TokenObject = token.isBNB ? KNSupportedTokenStorage.shared.busdToken : KNSupportedTokenStorage.shared.bnbToken
-    let otherTokenMatic: TokenObject = token.isBNB ? KNSupportedTokenStorage.shared.usdcToken : KNSupportedTokenStorage.shared.maticToken
+    let otherTokenMatic: TokenObject = token.isMatic ? KNSupportedTokenStorage.shared.usdceToken : KNSupportedTokenStorage.shared.maticToken
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/KyberSwap/SwapVC/Coordinator/KNExchangeTokenCoordinator.swift:161]: usdce for Matic seems incorrect

File: KyberNetwork/KyberNetwork/Screens/Tabbars/KyberSwap/SwapVC/Coordinator/KNExchangeTokenCoordinator.swift [hunk 5]
```diff
@@ -158,7 +158,8 @@ extension KNExchangeTokenCoordinator {
...
+    let otherTokenAvax: TokenObject = token.isAvax ? KNSupportedTokenStorage.shared.usdcToken : KNSupportedTokenStorage.shared.avaxToken
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/KyberSwap/SwapVC/Coordinator/KNExchangeTokenCoordinator.swift:162]: usdce?

**Review by toidv — CHANGES_REQUESTED** [review]

File: KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/Controllers/EarnMenuViewController.xib (hunk 3 above)
Comment (MrQKyber) [reply KyberNetwork/KyberNetwork/Screens/Tabbars/Earn/Controllers/EarnMenuViewController.xib:12]: Xcode auto change font in xib file. It is okay

**Review by toidv — APPROVED** [review]
LGTM

**Pull request merged**